docs/pages/includes/install-linux.mdx

```diff
@@ -1,220 +1,186 @@
+<Admonition type="tip">
+
+In the example commands below, update `$SYSTEM-ARCH` with the appropriate
+value (`amd64`, `arm64`, or `arm`). All example commands using this variable
+will update after one is filled out. This variable may not be present, depending
+on your installation type.
+
+Be sure to select the apporpriate scope at the top of this page (OSS,
```

Comment:
What would you think of replacing the `ScopedBlock`s with an outer `Tabs` component, but using the `dropdownView` property in the `Tabs` tag so it doesn't clash with the inner `Tabs` component? That way, the different scope options will always be visible:

```
<Tabs dropdownView dropdownCaption="Teleport Edition">
<TabItem label="Teleport Enterprise" scope="enterprise">
    <Tabs>
    </Tabs>
</TabItem>
<TabItem label="Teleport Enterprise Cloud" scope="cloud">
    <Tabs>
    </Tabs>
</TabItem>
<TabItem label="Teleport Community Edition scope="oss">
    <Tabs>
    </Tabs>
</Tabs>

Reply:
Updated in [0f4b599](https://github.com/gravitational/teleport/pull/17498/commits/0f4b5997d40d56830ce92f558efa1975058b317b)